xzcat: add page (#9022)
tldr-pages/tldr: pages/common/xzcat.md

# xzcat

> > A general purpose data compression tool similar to gzip and bzip2 syntaxes.

Reviewer: ```suggestion
> Data compression tool similar to `gzip` and `bzip2` syntax.
```

> > Compresses or decompresses .xz and .lzma files.

Reviewer: ```suggestion
> Compresses or decompresses `.xz` and `.lzma` files.
```

> > More information: <https://www.unix.com/man-page/linux/1/xzcat/>.

Reviewer: ```suggestion
> More information: <https://manned.org/xzcat>.
```

> - Force compress:

Reviewer: Perhaps a better explanation
```suggestion
- Compress a file then delete it:
```

`xzcat --compress {{path/to/file}}`

> - Force decompress:

Reviewer: ```suggestion
- Decompress a file:
```

> `xzcat --decopress {{path/to/file}}`

Reviewer: ```suggestion
`xzcat --decopress {{path/to/file.xz}}`
```

> - Test compressed file integrity:

Reviewer: ```suggestion
- Test the integrity of a specific archive:
```

> `xzcat --test {{path/to/file}}`

Reviewer: ```suggestion
`xzcat --test {{path/to/archive.xz}}`
```

> - List information about .xz file:

Reviewer: ```suggestion
- List the contents of a specific archive:
```

`xzcat --list {{path/to/file}}`

> - Keep/Avoid deleting input files:

Reviewer: ```suggestion
- Compress a file without delete it:
```

`xzcat --keep {{path/to/file}}`

> - Compress a file overwriting the output archive and decompress links:

Reviewer: @SethFalco could you please help here?
```suggestion
- Compress a file overwriting the output archive compressing the target of symbolic links:
```
